_posts/0000-01-02-dkwekPHX.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
**changing line 5 text**

Reply:
layout: slide
title: "Welcome to our second slide!"
---
my text changed here!